### ISSUE TITLE
skinport bug with headcrumbs and morph mod

### ISSUE THREAD

**Issue by vinicius1222** [zlainsama/SkinPort#16]
In version 6, the mod works well in rendering with the mod Headcrumbs (spiral lucky block). But from version 7 onwards the skin gets buggy, the same happens with morph mod in multiplayer. Is there any way to fix it?

**Comment by zlainsama** [comment]
Can you describe how buggy it is in more detail? something like texture messed up.
A screenshot would be helpful too.
It's been a long time since i last worked on this mod.
From version 6 to version 7, it seems those changes does not have anything to do with actual rendering.

**Comment by vinicius1222** [comment]
it is as if the skin did not look very well rendered, similar to version 1.6 when using the skin 64x64. But it only affects those mods. Headcrumbs 1.7.4
![Uploading 2019-02-07_14.38.51.png…]()

**Comment by zlainsama** [comment]
Does it also rendered weird in players? or is it only player heads?

**Comment by vinicius1222** [comment]
Only on the player heads. The player's rendering is normal.

**Comment by zlainsama** [comment]
The reason it renders weird is SkinPort doesn't have support for skulls to render a 64x64 skin.
If you are using Headcrumbs, you can try Et futurum, they are made by the same author.
Et futurum brings 1.8 features into 1.7, and it also has 64x64 skin support.
By digging through Headcrumbs' source code a little bit, i found out it has support for Et futurum.

**Comment by vinicius1222** [comment]
Yes, it would be of great help. This mod corrects the multiplayer bug. But anyway, thank you.

**Comment by zlainsama** [comment]
I released 1.7.10-v9a, it added support for vanilla playerheads to display properly, and other features from recently updated OfflineSkins.
Unfortunately, only partial support for Headcrumbs has been implemented, and i didn't test with Morph.

**Comment by vinicius1222** [comment]
Ok, thank you very much
